DuckDuckGo/DuckPlayer/YouTubePlayerNavigationHandler.swift

```diff
@@ -94,6 +99,17 @@ final class YoutubePlayerNavigationHandler {
         performNavigation(duckPlayerRequest, responseHTML: html, webView: webView)
     }
     
+    // Re-enables DP if required
+    private func updateTemporaryState(_ url: URL?) {
```

Comment:
DP can now be temporarily disabled for specific cases (Such as Watch in Youtube) scenarios.   The basic logic is that if the video you're about to watch is the same video you're already watching, it should open in Youtube and not Duckplayer.